### PR TITLE
Migrate CI service from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by hexium310**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    strategy:
+      matrix:
+        php: [7.3, 7.4, 8.0]
+        libraries:
+          - laravel: ^8.0
+            testbench: ^6.0
+          - laravel: ^7.0
+            testbench: ^5.0
+          - laravel: ^6.0
+            testbench: ^4.0
```
Comment (mpyw) [inline .github/workflows/ci.yml]: Write them in one line 

```suggestion
          - { laravel: ^8.0, testbench: ^6.0, composer_flags: --prefer-stable }
          - { laravel: ^7.0, testbench: ^5.0, composer_flags: --prefer-stable }
          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable }
          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable --prefer-lowest }
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.3, 7.4, 8.0]
+        libraries:
```
Comment (mpyw) [inline .github/workflows/ci.yml]: Simply `"lib"`

```suggestion
        lib:
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: xdebug
+
+      - run: composer require "laravel/framework:${{ matrix.libraries.laravel }}" "orchestra/testbench:${{ matrix.libraries.testbench }}" --no-update
+      - run: composer require "phpunit/phpunit:^9" --no-update
+      - run: composer update
```
Comment (mpyw) [inline .github/workflows/ci.yml]:
```suggestion
      - run: composer require "laravel/framework:${{ matrix.lib.laravel }}" "orchestra/testbench:${{ matrix.lib.testbench }}" --dev --no-update ${{ matrix.lib.composer_flags }}
      - run: composer require "phpunit/phpunit" --dev --no-update
      - run: composer update ${{ matrix.lib.composer_flags }}
```

**Review by mpyw — COMMENTED** [review]
Please also update README.md

**Comment by hexium310** [comment]
tests: https://github.com/hexium310/scoped-auth/actions/runs/972504253

**Review by mpyw — APPROVED** [review]

**Pull request merged**